packages/@webex/plugin-meetings/src/breakouts/breakout.ts

```diff
@@ -60,33 +58,21 @@ const Breakout = WebexPlugin.extend({
    */
   async join() {
     const breakoutMoveId = uuid.v4();
+    const deviceUrl = this.webex?.internal?.device?.url;
     const {meetingId} = this.collection.parent;
-    Metrics.postEvent({
-      event: eventType.MOVE_TO_BREAKOUT,
-      meetingId,
-      data: {
-        breakoutMoveId,
-        breakoutSessionId: this.sessionId,
-        breakoutGroupId: this.groupId,
-      },
-    });
+    const meeting = this.webex?.meetings?.getMeetingByType(_ID_, meetingId);
```

Comment:
issue: not of these question marks should be necessary. webex, the meetings plugin and getMeetingByType should always be defined.

Reply:
fixed